### DISCUSSION TITLE
Hologram Text Scaling

### DISCUSSION THREAD

**Discussion by mercurialmusic** [Zrips/CMI#8858] [Enhancement]
Not sure how feasible this is but have seen some other plugins with it, to a point where staff are asking me to switch. But it would be cool if hologram text could also be scalable.

**Comment by Zrips** [comment]
You can, if you are using new display method introduced with 1.19.4 version which gives you allot more options when it comes to hologram visualization.

**Reply by JaydenJinx** [reply]
Can confirm. Text scaling worlds perfectly fine for me.

**Comment by mercurialmusic** [comment]
Yup. You totally can. I'm blind. My apologies, and thanks!
